doc/index.rst

```diff
@@ -291,6 +279,33 @@ If pull requests should be checked against the merge parent instead of the stabl
     ABICHECK_URL='github:ros-industrial/ros_canopen#kinetic'
     ABICHECK_MERGE=auto
 
+URL can be specified in shortcut form `provider:organization/repository#version`, which is supported for bitbucket, github and gitlab. "`version`" can be either the name of the branch or the tagged version:
+
+- github:ros-industrial-release/ros_canopen-release#upstream           
+- gitlab:ipa-mdl/ci-example#master
+- github:ros-planning/moveit#0.9.9
+  
+Alternatively you can use the following forms as URL.:
+
+- https://github.com/ros-industrial/ros_canopen/archive/kinetic.zip
+- https://github.com/ros-industrial-release/ros_canopen-release/archive/upstream.zip
+- https://gitlab.com/ipa-mdl/ci-example/repository/master/archive.zip
+- https://github.com/ros-planning/moveit/archive/0.9.9.tar.gz
+
+With this format, archive needs to be pointed for branches. E.g. on GitHub, URL for a branch's archive can be https://github.com/organization/repository/archive/branch.zip
```

Comment:
I really like this structure!
I misunderstood "archive needs to be pointed for branches" first.

"With this format, the URL needs to point to an actual archive. E.g. on GitHub [...]" ?